hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/Http2ClientParameters.java

```diff
@@ -0,0 +1,42 @@
+package pl.allegro.tech.hermes.consumers.consumer.sender.http;
+
+public class Http2ClientParameters {
+
+    private final boolean enabled;
```

Comment:
As I can see this is an example of property used only in configuration package. Please let me know if I'm wrong, but this one can be removed from this class. We don't use this inside domain package.